src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -73,7 +77,6 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
                         switchAccessibilityLabel={translate('workspace.qbo.defaultVendorDescription')}
                         wrapperStyle={[styles.ph5, styles.mb3, styles.mt1]}
                         isActive={!!qboConfig?.autoCreateVendor}
-                        shouldPlaceSubtitleBelowSwitch
```

Comment:
Looks like `shouldPlaceSubtitleBelowSwitch` prop was removed during a bad merge, and that caused the issue https://github.com/Expensify/App/issues/59479